projects/batfish/src/org/batfish/grammar/cisco/Cisco_common.g4

```diff
@@ -18,6 +18,7 @@ community
    | com = LOCAL_AS
    | com = NO_ADVERTISE
    | com = NO_EXPORT
+   | com = GSHUT
```

Comment:
I haven't seen your config, but I'm fairly certain GSHUT is a variable and not a well-known community. I don't think it belongs here, nor do I believe that the lexer rule for GSHUT should exist.